### PR TITLE
latex-workshop extension

### PR DESCRIPTION
**Pull request by The-Ludwig**
I put a very extensive explanation on what "latex-workshop" can do into the blog. 

And: I deleted an unnecessary assign to category in liquid-code in _layouts/category.html.
It was "{% assign category = site.data.categories[page.category] %}" but site.data.categories does not even exist (probably site.categories was meant). I checked it, it returns an empty variable, so I just deleted it, and everything still works fine. Hopefully there wasn't any bigger meaning behind this (It could produce an error, if somebody checked if the variable exists (because it is nil or something now))

### CODE REVIEW

**Comment by MaxNoe** [comment]
I think this was just a leftover from the pep main page, where we split stuff into categories defined in a data.yaml

File: blog/_posts/2019-10-07-vs-code-latex-extension.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Diese kann Syntax Highlighting (Passende Einfärbung des LaTeX-Textes, für die Übersichtlichkeit), Snippets (Kleinere LaTeX-Code-Auszüge, die man häufig verwendet, sodass man diese nicht andauernd tippen muss, sondern z.B. mit `str`+`leertaste` snippets auswählen kann) und folding (z.B. Verstecken einer Mathe-Umgebung, für die Übersichtlichkeit).
+
+Es gibt allerdings eine noch viel mächtigere Extension, die euch die Arbeit mit LaTeX in vscode erleichtert. Diese heißt `LaTex Workshop` von James Yu. Dieses kann alles, was auch `LaTeX language support` kann, jedoch zusätlich noch: PDFs anzeigen, code-completion mit intellisense 
+(zeigt dir an, was du mit deinem angefangenen code meinen könntest, z.B.: tippe `\autoref{fig:` -> zeigt dir alle Referenz mit `fig:` beginnend an.) und es zeigt dir sogar eine Vorschau der Mathe-Sachen die du gerade tippst (FYI: mit [MathJax](https://www.mathjax.org/)). 
+Das ganze Add-On kann noch vieles mehr, und wer interessiert ist, sollte sich definitiv mit dem ganzen genauer beschäftigen (Stichworte [SynchTex](https://github.com/James-Yu/LaTeX-Workshop/wiki/View) und [Linting](https://github.com/James-Yu/LaTeX-Workshop/wiki/Linters)).
```
Comment (MaxNoe) [inline blog/_posts/2019-10-07-vs-code-latex-extension.md]: SyncTex

File: blog/_posts/2019-10-07-vs-code-latex-extension.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Da wir jedoch immernoch die im Toolbox-Workshop gelernte Toolchain benutzen wollen, müssen wir die Extension daran hindern, dass sie die LaTeX-Dokumente selber baut. (Es ist auch möglich, die Extension so zu konfigurieren, dass sie die genau so baut, wie ihr das auch machen würdet, immer wenn ihr abspeichert. Das müsst ihr aber selbst herausfinden. &#x1f61c;)
```
Comment (MaxNoe) [inline blog/_posts/2019-10-07-vs-code-latex-extension.md]: How about unicode?

File: blog/_posts/2019-10-07-vs-code-latex-extension.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- In den Einstellungen muss jetzt noch das automatische Kompilieren der Extension ausgestellt werden, damit die Vorlagen des Toolbox-Workshop auch problemlos bauen.
+- Dazu in die `Settings` gehen (`File`->`Preferences`->`Settings` oder Zahnrad links unten anklicken und `Settings` wählen.)
+- `latex auto build clean and retry` suchen und Häkchen entfernen
+- `latex auto build run` suchen und im Drop-Down-Menü `never` einstellen 
+
+Jetzt seid ihr dem [WYSIWYG](https://en.wikipedia.org/wiki/WYSIWYG) sehr nahe!
+
+Hier noch zwei Extensions, die euch das Leben erheblich vereinfachen: 
+- `Code Spell Checker` mit `German - Code Spell Checker` beides von `Street Side Software` - Rechtschreibprüfung
+- `Python` von `Microsoft` - Ähnlich mächtig wie LaTeX Workshop nur für Python
```
Comment (MaxNoe) [inline blog/_posts/2019-10-07-vs-code-latex-extension.md]: Note, maybe make a post about this?

File: blog/_posts/2019-10-07-vs-code-latex-extension.md (hunk 1 above)
Comment (The-Ludwig) [inline blog/_posts/2019-10-07-vs-code-latex-extension.md]:
```suggestion
Das ganze Add-On kann noch vieles mehr, und wer interessiert ist, sollte sich definitiv mit dem ganzen genauer beschäftigen (Stichworte [SyncTex](https://github.com/James-Yu/LaTeX-Workshop/wiki/View) und [Linting](https://github.com/James-Yu/LaTeX-Workshop/wiki/Linters)).
```

File: blog/_posts/2019-10-07-vs-code-latex-extension.md (hunk 2 above)
Comment (The-Ludwig) [reply blog/_posts/2019-10-07-vs-code-latex-extension.md]: I like to write what's directly on my keyboard... but ok...
```suggestion
Da wir jedoch immernoch die im Toolbox-Workshop gelernte Toolchain benutzen wollen, müssen wir die Extension daran hindern, dass sie die LaTeX-Dokumente selber baut. (Es ist auch möglich, die Extension so zu konfigurieren, dass sie die genau so baut, wie ihr das auch machen würdet, immer wenn ihr abspeichert. Das müsst ihr aber selbst herausfinden. 😜)
```

File: blog/_posts/2019-10-07-vs-code-latex-extension.md (hunk 3 above)
Comment (The-Ludwig) [reply blog/_posts/2019-10-07-vs-code-latex-extension.md]: Good Idea... will do that some time next week.

File: blog/_posts/2019-10-07-vs-code-latex-extension.md (hunk 2 above)
Comment (MaxNoe) [reply blog/_posts/2019-10-07-vs-code-latex-extension.md]: Code is more often read than written.  Now I can see it's a nice smiley, before I had to google the escape sequence.

**Comment by MaxNoe** [comment]
For the future, make sure to branch off of the current master before doing any changes, e.g. like this:

```
$ git fetch
$ git checkout -b foo origin/master # new branch starting at origin/master
```

**Pull request merged**